docs/user/dql/functions.rst

```diff
@@ -1434,20 +1434,21 @@ Description
 
 Usage: dayofweek(date) returns the weekday index for date (1 = Sunday, 2 = Monday, …, 7 = Saturday).
 
+The `day_of_week` function is also provided as an alias.
```

Comment:
do we need an underscore to use this as an alias?

Reply:
Yes you do need the underscores. Both `dayofweek` and `day_of_week` work, but the version with the underscores is an alias to `dayofweek` which already exists and is implemented. However, `day_ofweek` and `dayof_week` do not work if that is what you are trying to ask.